src/overlord/Proxy.js

```diff
@@ -438,6 +438,11 @@ export class ProxyOverlord {
         console.log("Proxy finished any pending caching transactions");
     }
 
+    async finishRockHeaderUpdating() {
+        await this._remoteCall("/finishRockHeaderUpdating");
```

Comment:
To stay in sync with https://github.com/measurement-factory/squid-overlord/pull/4#pullrequestreview-1665048575 changes, this code will need to mimic finishStagingRequests() code:

```suggestion
        const options = {
            'job.type': '\bRock::HeaderUpdater',
        };
        await this._remoteCall("/finishJobs", options);
```

I hope `\b` part of the regex will be relayed correctly, but I do not know that.